src/modules/missionHelper/i18n/nb_NO.json

```diff
@@ -0,0 +1,93 @@
+		"possible_patient_exact": "kun {n} pasient | kun {n} pasienter",
+		"possible_patient_min": "minimum pasienter",
+		"specializations": "Utdanning/Spesialitet",
+		"title": "Pasienter | {n} pasient | {n} Pasienter"
+	},
+	"prerequisites": {
+		"brush_extension": "Forestry Expansion | Forestry Expansions",
+		"federalpolice_stations": "Federal Police Station | Federal Police Stations",
+		"fire_aviation_count": "Firefighting Plane Station | Firefighting Plane Stations",
+		"fire_stations": "Fire Station | Fire Stations",
+		"max_police_stations": "Maximum Police Stations",
+		"max_rescue_stations": "Maximum Rescue Stations",
+		"police_helicopter_stations": "Police Helicopter Station | Police Helicopter Stations",
+		"police_stations": "Police Station | Police Stations",
+		"rescue_stations": "Ambulance Station | Ambulance Stations",
+		"title": "Prerequisite | Prerequisites"
+	},
+	"prisoners": {
+		"max_possible_prisoners": "Maksimalt antall fanger",
+		"title": "Fange | {n} fange | {n} fanger"
+	},
+	"requirements": {
+		"water_needed": "Water demand (gal)"
+	},
+	"vehicles": {
+		"captions": {
+			"ambulances": "Ambulanse | Ambulanser",
+			"arff": "Flyplassbrannbil",
+			"battalion_chief_vehicles": "Innsatsleder brann | Innsatsleder brann",
+			"boats": "Båt | Båter",
+			"brush_truck": "Skogbrannbil | Skogbrannbil",
+			"fbi": "FBI Unit | FBI Units",
+			"fbi_bomb": "FBI Bomb Technician Vehicle | FBI Bomb Technician Vehicles",
+			"fbi_drone": "FBI Surveillance Drone | FBI Surveillance Drones",
+			"fbi_investigation": "FBI Investigation Wagon | FBI Investigation Wagons",
+			"fbi_mcc": "FBI Mobile Command Center Vehicle | FBI Mobile Command Center Vehicles",
+			"fire_aviation": "Airborne Firefighting | Airborne Firefightings",
+			"firetrucks": "Mannskapsbil | Mannskapsbiler",
+			"hazmat_vehicles": "CBRNe | CBRNe",
+			"heavy_rescue_vehicles": "Tungredningsbil | Tungredningsbil",
+			"helicopter": "Luftambulanse",
+			"k9": "Hundepatrulje",
+			"large_fire_boat": "Stor brannbåt | Store brannbåter",
+			"large_rescue_boat": "Stor redningsbåt | Store redningsbåter",
+			"mobile_air_vehicles": "Røykdykkerbil | Røykdykkerbiler",
+			"mobile_command_vehicles": "Innsatsstøttebil | Innsatsstøttebiler",
+			"platform_trucks": "Stigebil | Stigebiler",
+			"police_cars": "Patruljebil | Patriljebiler",
+			"police_helicopters": "Politihelikopter | Politihelikoptre",
+			"sheriff": "Utrykningspolitibil | Utrykningspolitibiler",
+			"swat_armored_vehicle": "Delta kjøretøy | Delta kjøretøy",
+			"swat_suv": "Panstret kjøretøy | Pansdrede kjøretøy",
+			"water_tankers": "Tankbil | Tankbiler"
+		},
+		"multifunctionals": {
+			"battalion_chief_vehicles": {
+				"additional_text": "",
+				"reduce_from": "mobile_command_vehicles"
+			},
+			"heavy_rescue_vehicles": {
+				"additional_text": "thereof at least. {n} Rescue Engines",
+				"reduce_from": "firetrucks"
+			}
+		},
+		"optional_alternatives": {
+			"allow_arff_instead_of_lf": {
```

Comment:
not in nb_NO

Reply:
Please translate the rw instread of lf. It's: "Tungredningsbil or  Mannskapsbil"